src/wp-content/themes/twentyseventeen/searchform.php

```diff
@@ -21,7 +21,7 @@
 			?>
 		</span>
 	</label>
-	<input type="search" id="<?php echo $unique_id; ?>" class="search-field" placeholder="<?php echo esc_attr_x( 'Search &hellip;', 'placeholder', 'twentyseventeen' ); ?>" value="<?php echo get_search_query(); ?>" name="s" />
+	<input type="search" id="<?php echo $unique_id; ?>" class="search-field" placeholder="<?php echo esc_attr_x( 'Search &hellip;', 'placeholder', 'twentyseventeen' ); ?>" value="<?php echo esc_attr( get_search_query() ); ?>" name="s" />
```

Comment:
`get_search_query()` already use [`esc_attr`](https://github.com/WordPress/wordpress-develop/blob/6.5/src/wp-includes/general-template.php#L4304-L4306) by default on the return value.

Reply:
No, that doesn't seem necessary. I had figured that the authors of about [850 themes](https://wpdirectory.net/search/01HYY02MJRZ5Z0GXDR557F2ZBB) knew something I didn't.

That also makes me question whether `esc_html()` would add any value, but I will revert the `esc_attr()` changes for now.

Reply:
Adding the `esc_html()` make sens to me.